util/fibers/detail/fiber_interface.cc

```diff
@@ -330,6 +316,35 @@ void FiberInterface::ExecuteOnFiberStack(PrintFn fn) {
   });
 }
 
+FiberInterface* FiberInterface::SwitchSetup() {
```

Comment:
refactored out as a separate function and reused in both SwitchTo and SwitchToAndExecute